jdaviz/configs/default/plugins/subset_plugin/subset_plugin.py

```diff
@@ -108,6 +115,13 @@ def _sync_available_from_state(self, *args):
                              self.subset_select._subset_to_dict(subset) for subset in
                              self.data_collection.subset_groups]
 
+    @observe('multiselect')
+    def _on_multiselect_change(self, change=None):
+        if not self.multiselect and isinstance(self.subset_selected, List):
+            self.subset_selected = self.subset_selected[0]
+        elif self.multiselect and isinstance(self.subset_selected, List):
+            self.is_centerable = True
```

Comment:
does any condition need to set this to False?

Reply:
I think it makes more sense in multiselect to loop through all selected options and send a snackbar message if one of the subsets failed. That way, the other subsets can still be re-centered and the user will know why a composite subset was not. So to answer your question, I think in multiselect it does not need to be False.

Reply:
Ok, so in single-select mode is there logic elsewhere that _might_ set `is_centerable = False`?  If so, maybe just a comment pointing to where that happens - on the first read I would have expected some other condition in this same block to set it to False.

Reply:
> so in single-select mode is there logic elsewhere that might set `is_centerable = False`

I think for historical reason, I disabled centering for composite Subset. It used to be not possible. Now it is possible after I patched glue-core but untested and not useful for Imviz.

Reply:
In multi-select mode, if *any* of the selected Subset is not center-able, the centering should be disabled. I cannot tell here if this is indeed what you coded up.

Reply:
If we disable in advance, then we need to add a message stating _which_ of the selected subsets are not eligible for recentering.  Alternatively, we could skip them when clicking recentering and display a snackbar message with the list that were skipped.

Reply:
This current implementation works as @kecnry described: "skip the [composite subset] when clicking recenter and display a snackbar message with the subset(s) that were skipped."

Reply:
To answer your original question @kecnry, `is_centerable` is set back to False each time `_unpack_get_subsets_for_ui` is called and the subset is composite:
https://github.com/spacetelescope/jdaviz/blob/96d974aeca77376349b40b533568acf00c3c60e7/jdaviz/configs/default/plugins/subset_plugin/subset_plugin.py#L139-L144